docs/v0.6/en/user-guide/client-libraries/go.md

```diff
@@ -239,25 +343,47 @@ m.DB = db
 
 {template query-library-raw-sql%
 
+#### Declare model
+
+```go
+type CpuMetric struct {
+    Host        string    `gorm:"column:host;primaryKey"`
+    Ts          time.Time `gorm:"column:ts;primaryKey"`
+    CpuUser     float64   `gorm:"column:cpu_user"`
+    CpuSys      float64   `gorm:"column:cpu_sys"`
+}
+```
+
+If you are using the ORM API to insert data, you can declare the model with both Gorm and Greptime tags.
+
 ```go
-type Monitor struct {
-	Host        string    `gorm:"column:host;primaryKey"`
-	Ts          time.Time `gorm:"column:ts;primaryKey"`
-	Cpu         float64   `gorm:"column:cpu"`
+type CpuMetric struct {
+    Host        string    `gorm:"column:host;primaryKey" greptime:"tag;column:host;type:string"`
+    Ts          time.Time `gorm:"column:ts;primaryKey"   greptime:"timestamp;column:ts;type:timestamp;precision:millisecond"`
+    CpuUser     float64   `gorm:"column:cpu_user"        greptime:"field;column:cpu_user;type:float64"`
+    CpuSys      float64   `gorm:"column:cpu_sys"         greptime:"field;column:cpu_sys;type:float64"`
 }
+```
 
-func (Monitor) TableName() string {
-	return "monitor"
+#### Declare table name
+
+```go
+func (CpuMetric) TableName() string {
+	return "cpu_metric"
 }
+```
 
-var monitor Monitor
-db.Raw("SELECT host, cpu, ts FROM users WHERE ts > ?", 1701360000000).Scan(&result)
+#### Query data
```

Comment:
Maybe Query can be replaced in Query sections or some other page, but not in `client-library`? For client-library is not responsible for query, and the query code maybe influenced by MySQL or PG ecosystem, not by ingester-go itself.

Reply:
As discussed before, we should write some documentation about querying data to help users choose lib and write simple query code.